lib/generators/avo/templates/locales/avo.uk.yml

```
@@ -0,0 +1,120 @@
+---
+uk:
+  avo:
+    resource_translations:
+      product:
+        save: "Зберегти продукт!"
+    action_ran_successfully: Дія успішно виконана!
+    actions: Дії
+    and_x_other_resources: і ще %{count} інших ресурсів
+    applied: застосовано
+    are_you_sure: Ви впевнені?
+    are_you_sure_detach_item: Ви впевнені, що хочете відокремити цей %{item}?
```

use `відкріпити`, not `відокремити` everywhere
use `відкріплено`, not `відокремлено` everywhere

```suggestion
    are_you_sure_detach_item: Ви впевнені, що хочете відкріпити цей %{item}?
```